### ISSUE TITLE
ABS Model

### ISSUE THREAD

**Issue by lukas-schott** [robust-ml/robust-ml.github.io#2]
**Name**: Towards the first adversarially robust neural network model on MNIST

**Authors**: Lukas Schott, Jonas Rauber, Matthias Bethge, Wieland Brendel

**Paper**: https://arxiv.org/pdf/1805.09190.pdf

**Code**: https://github.com/bethgelab/AnalysisBySynthesis

**Venue**: 2018 NeurIPS Security Workshop

**Does the code implement the robust-ml API and include pre-trained models**: yes

**Dataset**: MNIST

**Threat model**: L2 (\epsilon = 1.5)

**Natural accuracy**: 99%

**Claims**: 80%

**Comment by anishathalye** [comment]
This looks great -- thank you for the submission! I'll take a look at it and try to update the website sometime in the next few days (after NeurIPS).

**Comment by anishathalye** [comment]
Thanks again for the submission -- merged in ef26ecb8dbc53bc41235b1a80d2c8b6972990b2a!
